geo_heatmap.py

```diff
@@ -53,21 +53,37 @@ def loadJSONData(self, json_file, date_range):
         """
         data = json.load(json_file)
 
-        # Find the correct key for timestamps
-        first_element = data["locations"][0]
-        key_timestamp = self.findTimestampKey(first_element)
+        if 'semanticSegments' in data.keys():
+            w = [Bar(), Percentage(), " ", ETA()]
+            with ProgressBar(max_value=len(data["semanticSegments"]), widgets=w) as pb:
+                for i, dataEntry in enumerate(data["semanticSegments"]):
+                    if "timelinePath" in dataEntry.keys():
+                        for entry in dataEntry["timelinePath"]:
+                            # print(entry)
+                            latLon = entry["point"].replace("Â°", "").split(", ")
+                            coords = (float(latLon[0]), float(latLon[1]))

```

Comment:
Hi, the code didn't work for me until I changed this to `coords = (float(latLon[0][:-1]), float(latLon[1][:-1]))` - my JSON seems to add a degree symbol to the locations.